src/Microsoft.DotNet.Darc/Darc/Models/PopUps/EditorPopUp.cs

```diff
@@ -79,6 +79,12 @@ protected static string GetCurrentSettingForDisplay(string currentValue, string
     /// </returns>
     protected static string ParseSetting(string inputSetting, string originalSetting, bool isSecret)
     {
+        //if the setting is null, trimming will throw an exception
```

Comment:
> ConsoleOutput: fail: There was an exception processing YAML input from stdin

I'd try to investigate why the "YAML input" changed and we started to get exceptions

Reply:
It comes from the test, that's something new that Premek added: in the test the value of `failureNotificationTags` is not set but we're trying to parse it [link](https://github.com/dotnet/arcade-services/blob/f78fce70cdf7f2a91475f210ad77c18e60cb91ff/test/Maestro.ScenarioTests/ScenarioTests_Subscriptions.cs#L162)